### ISSUE TITLE
Feature Request: Support another case of using Promises.all()

### ISSUE THREAD

**Issue by daenamkim** [playwright-community/eslint-plugin-playwright#188]
Related to https://github.com/playwright-community/eslint-plugin-playwright/issues/107

It looks like I am asking how to refactor codes though 😅, I wonder if it is possible to make `CASE: ERROR` below passed.

<img width="640" alt="image" src="https://github.com/playwright-community/eslint-plugin-playwright/assets/16159767/11ed436f-1923-48e2-97f8-eae628666d54">

```sh
...
  48:21  error  'toBeHidden' must be awaited or returned   playwright/missing-playwright-await
  49:21  error  'toHaveCount' must be awaited or returned  playwright/missing-playwright-await

✖ 2 problems (2 errors, 0 warnings)
  2 errors and 0 warnings potentially fixable with the `--fix` option.

error Command failed with exit code 1.
```
